src/routes/hall_of_fame/+page.svelte

```diff
@@ -0,0 +1,198 @@
+<script>
+	import Title from '$lib/components/Title.svelte'
+	import Contest from './Contest.svelte'
+	import FamedRice from './FamedRice.svelte'
+	import colors from 'tailwindcss/colors'
+</script>
+
+<section>
+	<div class="hero-wrapper">
+		<div class="japanese" aria-hidden="true">
+			<div class="rice-symbol">「 米 」</div>
+			<div class="wisdoms">
+				'ベンソンハウス-デイアングルオン：地面にスニーカー。カメラが上に移動して、バリーベンソンのベッドルームアングルを表示します。バリーの手がクローゼットの中でさまざまなセーターをめくっています。バリーイエローブラック、イエローブラック、イエローブラック、イエローブラック、イエローブラック、イエローブラック...うーん、ブラックとイエロー...アングル：バリーが選んだセーターを着て、鏡を見て。バリー（続き）ええ、少し振りましょう彼は黒と黄色のものを選びます。彼はそれから流しに行き、蜂蜜の容器を外し、髪に蜂蜜を入れます。彼は口の中でいくつか潮吹きしてうがいをします。それから彼はボトルの蓋を外し、デオドラントのようにいくつかを転がします。カット'
+			</div>
+		</div>
+		<div class="gradient_" />
+		<Title>
+			<span slot="title" class="title">Hall of fame</span>
+			<div slot="subtitle" class="max-w-[40ch]">
+				The chronicles of the triumphant from bygone rice contests held within our Discord
+			</div>
+		</Title>
+	</div>
+
+	<Contest name="Space" number={3} date="November 2023" --color={colors.indigo[500]}>
+		<FamedRice
+			name="Celestial"
+			creator="Flafy"
+			pretitel="#1"
+			dotfilesLink="https://github.com/flafydev/nixos-config/"
+			creatorProfilePicture="https://avatars.githubusercontent.com/u/44374434?s=96&v=4"
+			thumbnail="/imgs/ricing_competitions/3/flafy.webp"
+		/>
+		<FamedRice
+			name="Globes"
+			creator="Aylur"
+			pretitel="#2"
+			dotfilesLink="https://github.com/Aylur/dotfiles"
+			creatorProfilePicture="https://avatars.githubusercontent.com/u/104676705?v=4"
+			thumbnail="/imgs/ricing_competitions/3/aylur.webp"
+		/>
+		<FamedRice
+			name="Golden Era"
+			creator="VDawg"
+			pretitel="#3"
+			dotfilesLink="https://github.com/Visual-Dawg/space_dots"
+			creatorProfilePicture="https://avatars.githubusercontent.com/u/28539403?s=96&v=4"
+			thumbnail="/imgs/ricing_competitions/3/vdawg.webp"
+		/>
+	</Contest>
+
+	<Contest name="Summer" number={2} date="May 2023" --color="orange">
+		<FamedRice
+			name="Summer Gruv"
+			creator="end_4"
+			pretitel="#1"
+			dotfilesLink="https://github.com/end-4/dots-hyprland/tree/summer-gruv"
+			creatorProfilePicture="https://avatars.githubusercontent.com/u/97237370?s=24&v=4"
+			thumbnail="/imgs/ricing_competitions/2/end_4.webp"
+		/>
+		<FamedRice
+			name="Unnamed"
+			creator="Flafy"
+			pretitel="#2"
+			dotfilesLink="https://github.com/FlafyDev/flutter_background_bar"
+			creatorProfilePicture="https://avatars.githubusercontent.com/u/44374434?s=96&v=4"
+			thumbnail="/imgs/ricing_competitions/2/flafy.webp"
+		/>
+		<FamedRice
+			name="Day and Night"
+			creator="Mathisbuilder"
+			pretitel="#3"
+			dotfilesLink="https://github.com/MathisP75/summer-day-and-night"
+			creatorProfilePicture="https://avatars.githubusercontent.com/u/98901170?v=4"
+			thumbnail="/imgs/ricing_competitions/2/day-night.webp"
+		/>
+	</Contest>
```

Comment:
I'd use a function that iterates over a json file to construct the FamedRice components here